driver/csiplugin/controllerserver.go

```diff
@@ -417,43 +395,55 @@ func (cs *ScaleControllerServer) CreateVolume(ctx context.Context, req *csi.Crea
 
 	// Check if Primary Fileset is linked
 	primaryFileset := cs.Driver.primary.PrimaryFset
-	glog.V(4).Infof("check if primary fileset [%v] is linked", primaryFileset)
+	glog.V(4).Infof("volume:[%v] - check if primary fileset [%v] is linked", scaleVol.VolName, primaryFileset)
```

Comment:
Whats your take on these debug messages for intermediate steps? Should they be V(5) or V(6)? Or maybe under Debugf? Or remove them altogether in the final delivered code?

Reply:
It is better to have for debug purpose. Agree that it shouldn't be always there so moved it to v5